osmnx/graph.py

```diff
@@ -44,7 +44,7 @@ def graph_from_bbox(
     simplify: bool = True,
     retain_all: bool = False,
     truncate_by_edge: bool = False,
-    custom_filter: str | None = None,
+    custom_filter: str | list[str] | None = None,
```

Comment:
You can update the docstring for each of these public-facing functions so the user knows the difference between providing a string vs a list of strings.

Reply:
@gboeing how about 
```
If single str is provided, intersection will be used to combine results, 
e.g `'[maxspeed=50][lanes=2]'` will query all ways with maximum 
speed of 50 and having two lanes.
If list[str], union will be used, e.g. `['[maxspeed=50]', '[lanes=2]']` 
will query all ways with maximum speed of 50 or having two lanes.
```

or without examples
```
If single str is provided, intersection will be used to combine results.
If list[str], union will be used.
```